spec/migrations/20170605090919_add_expression_type_to_miq_alert_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_migration
+
+describe AddExpressionTypeToMiqAlert do
+  migration_context :up do
+    let(:miq_alert_stub) { migration_stub(:MiqAlert) }
+
+    it 'alert with hash expression has expression_type Hash' do
+      hash_expression = {:key => 'value'}
+      alert = miq_alert_stub.create!(:description => 'Test Alert', :expression => hash_expression)
+
+      migrate
+
+      alert.reload
+      expect(alert.expression_type).to eq("Hash")
+    end
+
+    it 'alert with miq_expression has expression_type MiqExpression' do
+      miq_expression = YAML.load '--- !ruby/object:MiqExpression
```

Comment:
This forces the object to be serialized, which isn't necessary.  You can just write this string directly into the column if you don't serialize in the stub in the first place a la https://github.com/ManageIQ/manageiq/pull/15315/files#r121777567